### PR TITLE
Added test visual

### PR DESCRIPTION
**Pull request by gabtam55**
# Summary
This branch allows Gabriel to experiment adding Shiny visuals.

# Changes
The changes made in this PR are:
1. Test visuals added to the "Collision Location Map' tab
2. Section comments were added to make the script more organised

***

# Check
- [x] The visual was added
- [ ] The R scripts were edited according to our agreed workflow

### CODE REVIEW

**Comment by gabtam55** [comment]
Hi @avisionh, thanks for agreeing to check this for me. Comments are welcomed!

File: inst/app/global.R
Comment (avisionh) [inline inst/app/global.R:20]: Could you `renv::snapshot()` these packages, please?

File: inst/app/modules/plotly_testvisual.R
Comment (avisionh) [inline inst/app/modules/plotly_testvisual.R:7]: This is nice, Plotly has moved on since the last time I looked at it!

File: inst/app/server.R
Comment (avisionh) [inline inst/app/server.R:23]: Fantastic to see you using modules from the very start - it's really useful in terms of chunking up and isolating independent pieces of logic.

**Review by avisionh — CHANGES_REQUESTED** [review]
A great start @gabtam55, especially liking the use of modules from the start.

Just two small things to change before I approve:
- [ ] Can you delete the `.DS_Store` file?
- [ ] Could you record the versions of the packages you used, as per the review comment?

Some other things to consider, but they are not necessary to change for me to approve:
1. The text on the x-axis of the plot can overflow on top of each other on smaller screens. Some ways to get around this are to word-wrap or even align them at something like a 45 degree angle.
image
1. The git commit message for this was:
> Added test visual to "Collision Location Map' tab + Added section comment to organise the script better
It's a nice, succinct and informative commit message but can be more aligned to our [Git style guide](https://github.com/Hong-Kong-Districts-Info/hktrafficcollisions/wiki/House-Style:-Git), by tweaking to something like the below. It uses the prefix tag of `feat` to denote you're adding a feature, uses the present-tense and splits out the message into two lines, first line being the main change and second line providing further description of the changes, as well as the reasoning for why you made this change.
> feat: Add bar chart to 'Collision Location Map' tab
>
> This is so we can test how it looks on the dashboard. 
>
> Also add section comment to organise the script better

File: inst/app/global.R
Comment (gabtam55) [reply inst/app/global.R:20]: Yes! Just did so.

File: inst/app/modules/plotly_testvisual.R
Comment (gabtam55) [reply inst/app/modules/plotly_testvisual.R:7]: Gosh, did you mean Plotly won't be updated anymore? What substitute package do you use?

File: inst/app/server.R
Comment (gabtam55) [reply inst/app/server.R:23]: Thanks for your tips indeed!

**Comment by gabtam55** [comment]
> A great start @gabtam55, especially liking the use of modules from the start.
> 
> Just two small things to change before I approve:
> 
> * [x]  Can you delete the `.DS_Store` file?
> * [x]  Could you record the versions of the packages you used, as per the review comment?
> 
> Some other things to consider, but they are not necessary to change for me to approve:
> 
> 1. The text on the x-axis of the plot can overflow on top of each other on smaller screens. Some ways to get around this are to word-wrap or even align them at something like a 45 degree angle.
>    image
> 2. The git commit message for this was:
> 
> > Added test visual to "Collision Location Map' tab + Added section comment to organise the script better
> 
> It's a nice, succinct and informative commit message but can be more aligned to our [Git style guide](https://github.com/Hong-Kong-Districts-Info/hktrafficcollisions/wiki/House-Style:-Git), by tweaking to something like the below. It uses the prefix tag of `feat` to denote you're adding a feature, uses the present-tense and splits out the message into two lines, first line being the main change and second line providing further description of the changes, as well as the reasoning for why you made this change.
> 
> > feat: Add bar chart to 'Collision Location Map' tab
> > This is so we can test how it looks on the dashboard.
> > Also add section comment to organise the script better

Thanks a lot for the comments!

File: inst/app/modules/plotly_testvisual.R
Comment (avisionh) [reply inst/app/modules/plotly_testvisual.R:7]: Oh no, I mean plotly has gotten better since I last used it in R (at least a year ago). 😅

**Review by avisionh — APPROVED** [review]
Thanks for making the changes now - happy with them. 🙂 Just a final note, the git commit message associated to deleting the `.DS_Store` file would be under the heading of:

`chore: Delete unnecessary .DS_Store file`

**Comment by gabtam55** [comment]
> Thanks for making the changes now - happy with them. 🙂 Just a final note, the git commit message associated to deleting the `.DS_Store` file would be under the heading of:
> 
> `chore: Delete unnecessary .DS_Store file`

You know what, really appreciated these detailed comments! Thanks.

**Comment by martinctc** [comment]
Hey guys - is this still a draft PR? Justin and @chautinsum have created a new wireframe:
https://www.figma.com/file/h64sro83dNwzLMNWl2o4H1/Collisions?node-id=0%3A1

Would it make sense to merge this PR and then use a new PR to try to mirror that wireframe? Or we could carry on with this PR.

**Comment by avisionh** [comment]
> Hey guys - is this still a draft PR? Justin and @chautinsum have created a new wireframe:
> https://www.figma.com/file/h64sro83dNwzLMNWl2o4H1/Collisions?node-id=0%3A1
> 
> Would it make sense to merge this PR and then use a new PR to try to mirror that wireframe? Or we could carry on with this PR.

Very helpful. We merge this PR and then work with the new wireframe. Usually, the author of the PR will merge it when it has been approved by the reviewers and they're happy, so @gabtam55, feel free to merge and start working with the new wireframe when you're ready!

**Comment by gabtam55** [comment]
> > Hey guys - is this still a draft PR? Justin and @chautinsum have created a new wireframe:
> > https://www.figma.com/file/h64sro83dNwzLMNWl2o4H1/Collisions?node-id=0%3A1
> > Would it make sense to merge this PR and then use a new PR to try to mirror that wireframe? Or we could carry on with this PR.
> 
> Very helpful. We merge this PR and then work with the new wireframe. Usually, the author of the PR will merge it when it has been approved by the reviewers and they're happy, so @gabtam55, feel free to merge and start working with the new wireframe when you're ready!

Thanks guys. Let's merge it.

**Pull request merged**
